README.md

```diff
@@ -1,17 +1,31 @@
+# Google Cloud .NET Client
+.NET idiomatic client libraries for [Google Cloud Platform][cloud-platform] services.
+
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/gcloud-dotnet.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/gcloud-dotnet)
 
-Google Cloud .NET Client
-========================
+* [Homepage][language-landing-dotnet]
+* [API Documentation][api-reference-dotnet]
+
+This client supports the following Google Cloud Platform services:
 
-.NET client libraries for [Google Cloud Platform][cloud-platform] services.
+* [Google BigQuery](#google-bigquery)
+* [Google Cloud Datastore](#google-cloud-datastore)
+* [Google Cloud Logging](#google-cloud-logging)
+* [Google Cloud Pub/Sub](#google-cloud-pubsub)
+* [Google Cloud Storage](#google-cloud-storage)
 
 > Note: This client is a work-in-progress, and may occasionally
 > make backwards-incompatible changes. The layout of the repository
 > is likely to change, and names (of everything - assemblies,
 > namespaces, types, members...) should be regarded as provisional.
 
-Requirements
-------
+If you need support for other Google APIs, check out the [Google .NET API Client library][google-api-dotnet-client].
+
+## Example Applications
+
+* [getting-started-dotnet] - A [quickstart and tutorial][language-landing-dotnet] that demonstrates how to build a complete web application using Cloud Datastore, Cloud Storage, and Cloud Pub/Sub and deploy it to Google App Engine or Google Compute Engine.
```

Comment:
It only demonstrates deploying to Compute Engine for the time being, also Cloud SQL is included, so perhaps update the sentence to something like:
~~~
"..demonstrates how to build a complete web application using Cloud Datastore, Cloud SQL, Cloud Storage, and Cloud Pub/Sub and deploy it to Google Compute Engine."